q2_diversity/tests/test_beta.py

```diff
@@ -114,24 +115,30 @@ def test_beta_empty_table(self):
     def test_beta_phylogenetic(self):
         t = self.two_feature_table
         tree = self.three_feature_tree
-        actual = self.beta_phylogenetic(
-            table=t, phylogeny=tree, metric='unweighted_unifrac')
 
-        self.assertEqual(len(actual), 1)
+        for method, table_, tree_ in ((self.beta_phylogenetic, t, tree),
+                                      (self.beta_phylogenetic_meta, [t, t],
+                                       [tree, tree])):
```

Comment:
Commenting once, but this is applicable to all of these new implementation loops.

Perhaps we should use [`unittest.subTest`](https://docs.python.org/3.6/library/unittest.html#unittest.TestCase.subTest) to differentiate the tests in the runner report?

```suggestion
        for method, table_, tree_ in ((self.beta_phylogenetic, t, tree),
                                      (self.beta_phylogenetic_meta, [t, t],
                                       [tree, tree])):
                with self.subTest(method=method):
                     ...
```

Reply:
Did not know about `subTest`, that is super cool.